source/_components/history_graph.markdown

```diff
@@ -73,5 +73,18 @@ history_graph:
     # refresh: 0
 
 ```
+NOTE: In order to ensure that the graphs appear in the UI, you have to add the graphs that have been created in a new group in groups.yaml, the example below illustrates:
```

Comment:
This statement is actually incorrect. It doesn't have to be in `groups.yaml`, secondly, that is depended on your setup.

Reply:
Hi,

I followed the instructions on the instructions, and the graph did not appear, it only appeared after I placed it in groups.yaml, perhaps the rephraing could be done thus: 

"Depending on the the setup, the graphs might not appear in the UI, to ensure they do appear or if they do not appear, place them in groups.yaml as follows"


```
# prefereble name of graph should be descriptive, entities should be in snake_case notation

Graphs:
name: graphs
view: yes
icon: mdi:chart-line-variant
entities:
  - history_graph.lights_graph
  - history_graph.temperature
```

Reply:
That sounds like a solution, yes.